flair/datasets/relation_extraction.py

```diff
@@ -77,7 +74,7 @@ def __init__(
                 augment_train=augment_train,
             )
 
-        super(RE_ENGLISH_SEMEVAL2010, self).__init__(
+        super().__init__(
```

Comment:
These automatic super() call changes are somewhat surprising. I wonder what is the motivation for this?

Reply:
super() is sequivalent to super(TheNameOfTheCurrentClass, self) - so if you really are passing the current class name and self, then the shorter way to write it is chosen by ruff - it also makes it clear at first view that nothing extra ordinary with regards to super is going on. If you really want to e.g. skip one hierarchy on inheritance by doing super(TheNameOfASuperClass, self) [which would skip the TheNameOfASuperClass and search "above" it in the class hiarchy], then you would still pass the parameters to super, and a reviewer would see that you are really doing something special with super here.

Reply:
Ah ok, thanks for the info!